RealmSwift/Realm.swift

```diff
@@ -1101,6 +1101,25 @@ extension Realm {
     }
 }
 
+// MARK: Asymmetric Sync
+
+extension Realm {
+    /**
+     Creates an Asymmetric object, which will be synced unidirectionally and
+     cannot be queried locally, only objects which inherit from `AsymmetricObject`
+     can be created using this method.
+
+     Objects created using this method will not be added to the Realm.
+
+     - warning: This method may only be called during a write transaction.
+
+     - parameter object: The asymmetricObject to create.
+     */
+    public func create(_ object: AsymmetricObject) {
```

Comment:
This should be the same API as the normal `create()` function other than returning void.

Reply:
Do you mean instead of passing an Asymmetric Object, we pass the values? @tgoyne